crates/wasmtime/src/linker.rs

```diff
@@ -604,7 +617,15 @@ impl Linker {
     fn compute_imports(&self, module: &Module) -> Result<Vec<Extern>> {
         module
             .imports()
-            .map(|import| self.get(&import).ok_or_else(|| self.link_error(&import)))
+            .map(|import| {
+                self.get(&import)
+                    .or_else(|| {
+                        self.store
+                            .get_host_func(import.module(), import.name()?)
```

Comment:
Would it be possible to move this logic into `self.get` directly? That internally handles the module-linking feature where if you import an instance it can be satisfied with a function definition of a two-level import. I think that if `self.store` were consulted in `self.get` that it'd naturally support that though.

Reply:
I think you're right, `get` makes more sense for this implementation as it should behave as if the function were already defined in the linker even though it comes from the store.

I'll make that change.